config/covid_tracking_project.tf

```diff
@@ -0,0 +1,7 @@
+# Resources and routines for Covid Tracking Project ingestion.
```

Comment:
I think the data sources should be under a separate directory since we will be adding more of them. This means they will need to be in a separate terraform module and then called from the main module (https://www.terraform.io/docs/configuration/blocks/modules/syntax.html#calling-a-child-module).

In terms of input variables, I was thinking dataset names (and in the future routine names) dont necessarily need to be variables because they are qualified by project so if you are deploying the tf config in a new project there will be no conflict. I feel like our variable list is getting a bit unwieldy. What are your thoughts on this?

Reply:
I'm all for removing variables that we don't ever need to change. The only issue I can forsee is if someone wanted to have 2 deployments in their test project, which I have done to accomodate for running terraform both locally and through github actions (but that is very niche and hacky anyway). I'll get rid of the variables. We can always add them back later if necessary.